src/keymap/vi.rs

```diff
@@ -1198,6 +1199,32 @@ mod tests {
         assert_eq!(map.ed.cursor(), 0);
     }
 
+    #[test]
+    /// Cursor moves left when exiting insert mode.
+    fn vi_switch_from_insert_ctrl_lb() {
```

Comment:
Instead of duplicating this test with `ctrl-[`, replace some (or all) of the `Esc` presses in the original test with `ctrl-[`.